### ISSUE TITLE
Rule proposal: `html-element-name-kebab-casing`

### ISSUE THREAD

**Issue by ota-meshi** [vuejs/eslint-plugin-vue#499]
**Please describe what the rule should do:**

This rule enforces the tag name of the Vue component, HTML element, and custom element to be `kebab-case`.
It does not target elements of MathML or SVG.

**What category of rule is this? (place an "X" next to just one item)**

[X] Enforces code style
[ ] Warns about a potential error
[ ] Suggests an alternate way of doing something
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule will warn about:**

```html
<Div/>
<CustomElement/>
<VueComponent/>
```

**Why should this rule be included?**

see
https://github.com/vuejs/eslint-plugin-vue/pull/397#issuecomment-399195380

In order for us to implement the `component-name-in-template-casing` rule we need to distinguish between Vue components and non-Vue custom elements. But this is very difficult.

I think this rule(`html-element-name-kebab-casing`) will be useful for projects that prefer `kebab-case`.

`component-name-in-template-casing` https://github.com/vuejs/eslint-plugin-vue/issues/250

**Comment by michalsnik** [comment]
Closing in favor of #397
